### PR TITLE
catalog

### PR DESCRIPTION
**Pull request by SergeyShapovalenko**
- [DEMO LINK](https://SergeyShapovalenko.github.io/layout_catalog/)
- [TEST REPORT LINK](https://SergeyShapovalenko.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -3,10 +3,345 @@
...
+          <p class="card__price">
+            Price:
+            <span class="card__price-in-money">
+              $2,199
+            </span>
+          </p>
+          
+          <a 
+            class="card__button" 
+            href="BUY"
```
Comment (ViktorHavryliuk) [inline src/index.html:199]: If you click on link now it sends user to non-existing page 
```suggestion
            href="#" 
```

File: src/index.html [hunk 2]
```diff
@@ -3,10 +3,345 @@
...
+            BUY
+          </a>
+        </div>
+
+        <div class="card">
+          <img 
+          class="card__logo" 
+          src="images/imac.jpeg" 
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        >
```
Comment (ViktorHavryliuk) [inline src/index.html:315]:
```suggestion
          <img 
             class="card__logo" 
             src="images/imac.jpeg" 
             alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
           >
```

File: src/styles/blocks/card.scss [hunk 3]
```diff
@@ -0,0 +1,115 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 0 16px;
+  max-width: $card-windth;
```
Comment (ViktorHavryliuk) [inline src/styles/blocks/card.scss:5]: Also fix typo in variable's name
```suggestion
  width: $card-width;
```

File: src/styles/blocks/catalog.scss [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+ 
+  min-width: 1024px;
+  max-width: 1200px;
+}
+
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+
```
Comment (ViktorHavryliuk) [inline src/styles/blocks/catalog.scss:16]: remove second empty line

File: src/styles/blocks/header.scss [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+  padding: 0 50px;
+
+  height: 60px;
+
+  .home {
+    margin-top: 1px;
+
+    width: 40px;
+    height: 40px;
+
```
Comment (ViktorHavryliuk) [inline src/styles/blocks/header.scss:14]: Remove empty line before closing bracket

**Review by ViktorHavryliuk — CHANGES_REQUESTED** [review]
Good work, but there are some small suggestions, please fix those

File: src/styles/blocks/card.scss [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+  box-sizing: border-box;
+  margin-right: $card-margin-right;
+ 
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  flex-grow: 1;
+
+  &:nth-child(4),
+
+  &:nth-child(8) {
```
Comment (tetiana-ivanchenko) [inline src/styles/blocks/card.scss:15]:
```suggestion
  &:nth-child(4),
  &:nth-child(8) {
```

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]

**Review by SanyaBratashchuk — APPROVED** [review]

**Pull request closed without merging**